src/web/services/__snapshots__/session-result-csv.service.spec.ts.snap

```diff
@@ -426,6 +426,13 @@ Team,Recipient Name,Recipient's Email,Sub Question,Yes,No,Total,Average
 \\"Team 1.1</td></div>'\\"\\"\\",student4 In Course1,student4InCourse1@gmail.tmt,a) This student has done a good job.,0% (0) [1.25],0% (0) [-1.7],0,0
 \\"Team 1.1</td></div>'\\"\\"\\",student4 In Course1,student4InCourse1@gmail.tmt,b) This student has tried his/her best.,100% (1) [1.25],0% (0) [-1.7],1.25,1.25
 
+Per Recipient Statistics (Overall)
+Team,Recipient Name,Recipient's Email,Average,Breakdown
+\\"Team 1.1</td></div>'\\"\\"\\",\\"student1 In Course1</td></div>'\\"\\"\\",student1InCourse1@gmail.tmt,-0.44999999999999996,\\"1.25, -1.7\\"
+\\"Team 1.1</td></div>'\\"\\"\\",student2 In Course1,student2InCourse1@gmail.tmt,-0.44999999999999996,\\"-1.7, 1.25\\"
+\\"Team 1.1</td></div>'\\"\\"\\",student3 In Course1,student3InCourse1@gmail.tmt,2.5,\\"1.25, 1.25\\"
```

Comment:
Correct me if I'm wrong, but that `Average` column does not look like "Average". @moziliar (ref: #11196)

Reply:
No it is not... This is a bug from not dividing up the `total`. Will fix this bug soon.

Reply:
Ok will look into this... I need to locate the root cause of this particular case...
The rest seem to be following the logic of averaging all the breakdown values

Reply:
I think the fix needs to be against the release branch as it will have to be a hot fix for V7.